flight/Modules/System/systemmod.c

```diff
@@ -67,28 +65,61 @@
 #if defined(PIOS_SYSTEM_STACK_SIZE)
 #define STACK_SIZE_BYTES PIOS_SYSTEM_STACK_SIZE
 #else
-#define STACK_SIZE_BYTES 924
+#define STACK_SIZE_BYTES 1024
 #endif
 
-#define TASK_PRIORITY PIOS_THREAD_PRIO_LOW
+#define TASK_PRIORITY PIOS_THREAD_PRIO_NORMAL
+
+/* Generates an armed LED blink of 4.4Hz, close to the previous 5, and is
+ * a nice prime number to not generate as nasty beat frequencies as other
+ * choices */
+#define SYSTEM_UPDATE_PERIOD_MS 113
```

Comment:
How quickly it ran before was nondeterministic.  It ran faster when armed, before, to get faster LED blinks.  It also ran more if there were queue wakeups, etc.  IMO this is bad-- it could consume more resources based on unpredictable factors and the CPU use one sees on the bench unarmed would not be representative.

Now it always runs at the same speed.  It does run 9x faster when disarmed, and a little bit slower than it did before when armed.